ui/store/actions.js

```diff
@@ -1455,10 +1459,6 @@ export function updateMetamaskState(newState) {
       // progress.
       dispatch(initializeSendState({ chainHasChanged: true }));
     }
-    dispatch({
```

Comment:
For future reference: this change was needed so that the correct balance is used in the send state when recalculating the max. This ensures that `initializeSendState` has been called after the metamask state has been updated